engines/mus.cpp

```diff
@@ -146,25 +150,48 @@ namespace pono {
 
     /*
      * If the conjunct encodes a state update constraint (i.e. of form
-     * <stateVar>.next = ...), then uses that state variable's id for the
+     * <stateVar>.next = ...), then use that state variable's id for the
      * control var symbol. Otherwise use the conjunct's to_string as an
      * identifier.
      */
-    UnorderedTermMap transIdToConjunct;
+    unordered_map<string, Term> transIdToConjunct;
     for (auto &tc: transConjuncts) {
       Term id = tc;
       if (tc->get_op() == PrimOp::Equal) {
         Term lhs = *tc->begin();
         if (ts_.is_next_var(lhs)) {
-          id = lhs;
+          id = ts_.curr(lhs);
```

Comment:
What's the purpose of this?

Reply:
`curr` maps `Term`s  that represent the next state version of a variable to their current state counterpart e.g. `a1.next` -> `a1`. I switched to this so MUS elements appear as e.g. `TRANS_a1` instead of `TRANS_a1.next`.

Reply:
Got it, thanks.

Is this to make them easier to understand, or is it important for the regex matching below? If the former, I'd place this sort of change into its own commit (and then have the commit message say something similar to your comment above).